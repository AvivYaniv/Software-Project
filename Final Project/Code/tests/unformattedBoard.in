
	 	  
      	 	      
solve tests/b2_3_unformatted.txt
solve tests/b2_2_unformatted.txt
edit tests/b2_3_unformatted.txt
edit tests/b2_2_unformatted.txt