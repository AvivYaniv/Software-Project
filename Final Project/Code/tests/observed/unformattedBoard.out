Sudoku
------
Enter your command:
Enter your command:
Enter your command:
Enter your command:
---------------------------
|  1.  2.  3.|  4.  5.  6.|
|  4.  5.  6.|  1.  2   3 |
---------------------------
|  2.  1.  4.|  6.  3.  5.|
|  3.  6.  5.|            |
---------------------------
|  5.  4.  1 |            |
|  6.        |            |
---------------------------
Enter your command:
-----------
|  1.|  2 |
|  3*|  4*|
-----------
Puzzle solution erroneous
Enter your command:
---------------------------
|  1   2   3 |  4   5   6 |
|  4   5   6 |  1   2   3 |
---------------------------
|  2   1   4 |  6   3   5 |
|  3   6   5 |            |
---------------------------
|  5   4   1 |            |
|  6         |            |
---------------------------
Enter your command:
-----------
|  1 |  2 |
|  3*|  4*|
-----------
Enter your command:
Exiting...
